src/test/java/org/dz/solution/task33/Task33Test.java

```diff
@@ -0,0 +1,66 @@
+package org.dz.solution.task33;
+
+import org.junit.Test;
+
+import java.io.*;
+import java.util.*;
+
+
+public class Task33Test {
+    private static final String SOURCE_DATA_FILE = "src/main/resources/org/dz/solution/task33/MainFile";
+    private static final String RESULT_FILE = "src/main/resources/org/dz/solution/task33/OutputFile";
+    private static final String[] SYMBOLS = new String[]{",", "-", "\\.", ";", "(", ")", ":", "\"", "«", "»", "„", "“"};
+    public static final String WORD_DELIMITER = " ";
+
+    @Test
+    public void test() {
+    }
+
+    private void calculateSkippedNumbers() {
+        List<String> textFromFile = ReadingAFile(SOURCE_DATA_FILE);
```

Comment:
ReadingAFile -is a bad name.
1) Methods names must start with lower case letters. 
2) Don't use articles
3) The name has to reflect exactly what the method do.

Reply:
textFromFile - is a bad name for a variable. It has to reflect the content of the list.